app/views/layouts/_content.html.haml

```diff
@@ -63,6 +63,9 @@
             = render :partial => "layouts/listnav"
           = yield :left
         #custom_left_cell
+- elsif @angular_app_name
+  .container-fluid{:style => "overflow: hidden; height: 100%;", 'data-ng-app' => "#{@angular_app_name}"}
+    %div{'ui-view' => ''}
 - else
```

Comment:
@himdel Also noticed that ```ui-view``` is being used in ```_content.html.haml``` whereas I have it in my specific Angular form.
At the moment it feels like ```_content.html.haml``` should be left alone. We'll see.